test/fakes/fakedatastore/fakedatastore.go

```diff
@@ -33,10 +33,10 @@ type DataStore struct {
 
 var _ datastore.DataStore = (*DataStore)(nil)
 
-func New(tb testing.TB) *DataStore {
+func New(tb testing.TB, eventsBasedCache bool) *DataStore {
 	log, _ := test.NewNullLogger()
 
-	ds := sql.New(log, false)
```

Comment:
Let's just set this to true since all it influences is whether or not the events table gets populated based on database changes.